macros/utils/table_operations/seed.sql

```diff
@@ -0,0 +1,20 @@
+{% macro seed_elementary_model(relation, csv_path) %}
```

Comment:
cosmetics - seed has a different meaning in dbt. I'd rename to upload_csv_to_relation